src/plugins/data_source_management/public/plugin.ts

```diff
@@ -4,25 +4,35 @@
  */
 
 import { CoreSetup, CoreStart, Plugin } from '../../../core/public';
-import { DataSourceManagementPluginStart, DataSourceManagementSetupDependencies } from './types';
 
 import { PLUGIN_NAME } from '../common';
 
-const IPM_APP_ID = 'dataSources';
+import { ManagementSetup } from '../../management/public';
+import { UrlForwardingSetup } from '../../url_forwarding/public';
+
+export interface DataSourceManagementSetupDependencies {
+  management: ManagementSetup;
+  urlForwarding: UrlForwardingSetup;
+}
+
+const DSM_APP_ID = 'dataSources';
 
 export class DataSourceManagementPlugin
-  implements Plugin<void, DataSourceManagementPluginStart, DataSourceManagementSetupDependencies> {
-  public setup(core: CoreSetup, { management }: DataSourceManagementSetupDependencies) {
+  implements Plugin<void, void, DataSourceManagementSetupDependencies> {
+  public setup(
+    core: CoreSetup,
+    { management, urlForwarding }: DataSourceManagementSetupDependencies
+  ) {
     const opensearchDashboardsSection = management.sections.section.opensearchDashboards;
 
     if (!opensearchDashboardsSection) {
       throw new Error('`opensearchDashboards` management section not found.');
```

Comment:
I don't see any reason for using `` string formatter

Reply:
that code has been refactored